README.md

```diff
@@ -672,6 +672,16 @@ This environment variable must be set for both the `aws-vpc-cni-init` and `aws-n
 
 Note that enabling/disabling this feature only affects whether newly created pods have an IPv6 interface created. Therefore, it is recommended that you reboot existing nodes after enabling/disabling this feature. Also note that if you are using this feature in conjunction with `ENABLE_POD_ENI` (Security Groups for Pods), the security group rules will NOT be applied to egressing IPv6 traffic.
 
+#### `ENABLE_V4_EGRESS` (v1.15.1+)
+
+Type: Boolean as a String
+
+Default: `true`
+
+Specifies whether PODs in an IPv6 cluster support IPv4 egress. If env is set to `true`, range `169.254.172.0/22` is reserved for IPv4 egress. When enabled, traffic egressing an IPv6 pod destined to an IPv4 endpoint will be SNAT'ed via the node IPv4 address.
+
+Note that enabling/disabling this feature only affects whether newly created pods have an IPv4 interface created. Therefore, it is recommended that you reboot existing nodes after enabling/disabling this feature. Also note that if you are using this feature in conjunction with `ENABLE_POD_ENI` (Security Groups for Pods), the security group rules will NOT be applied to egressing IPv4 traffic.
```

Comment:
I don't think we need to add the part about SGPP as we don't support SGPP in IPv6 mode yet.

Reply:
I figured it was fine to keep that there for when future support arrives

Reply:
Should we have the env variable name as ``ENABLE_V4_EGRESS_IN_V6` ? 

because the context matters.

If we specify the context, then statement of SGPP can be omitted. Don't feel strongly about mentioning it here.

Reply:
Yeah, I chose this name because we have `ENABLE_V6_EGRESS` for IPv4 clusters, so I wanted to keep the naming the same

Reply:
> ENABLE_V6_EGRESS for IPv4 clusters

For some reason, the IPV4 clusters are considered 'default' and that made immediate sense to me.   When I read `ENABLE_V4_EGRESS` I was thinking what I am missing, and then realized the context.

Keeping it consistent sounds good.

Reply:
It will not be an issue even when we support it because the SG is applied against a branch ENI in SGPP mode (and not pod's interfaces) and no matter which interface a packet takes to exit out of the pod it will have to go out of the branch ENI and so will be evaluated against the SG the customer provides and is configured on branch ENIs.

Reply:
Ah that makes sense, I am not sure why I originally had that line for `ENABLE_V6_EGRESS` then. Maybe because I was trying to point out that IPv4 policies would not be applied to IPv6 traffic and so you need IPv6-specific policies. Anyway, updated in the latest revision